transfer/src/main/java/org/kinecosystem/transfer/receiver/view/AccountInfoActivityBase.java

```diff
@@ -66,12 +66,18 @@ public void close() {
     }
 
     @Override
-    public void updateSourceApp(String sourceApp) {
+    public void updateTransactionInfo(String senderAppName, String memo, String receiverAppId , String senderAppId) {
+        //update view
         TextView title = findViewById(R.id.transfer_title);
         final CharSequence destinationApp = getApplicationInfo().loadLabel(getPackageManager());
-        title.setText(getString(R.string.receiver_activity_message, sourceApp, destinationApp));
+        title.setText(getString(R.string.receiver_activity_message, senderAppName, destinationApp));
+        //update receiver activity
+        onTransactionInfoReceived(senderAppName, formatFullMemo(memo, receiverAppId), senderAppId);
```

Comment:
let's remove this - anyone needing this information can override the 'updateTransactionInfo" method